### PR TITLE
introduce page done

### PR DESCRIPTION
**Pull request by lazy-sky**
Can I get a review for my codes? Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
-	<body></body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Introduce</title>
+  </head>
+  <body>
+    <h1>김하늘의 자기소개입니다.</h1>
+    <p>안녕하세요! 저는 프론트엔드 개발자를 꿈꾸는 김하늘입니다. 만나서 반갑습니다.</p>
+    <ul> 앞으로 공부할 것
```
Comment (imparang) [inline index.html:11]: Could you try the ol tag instead of the ul tag?

File: index.html [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
-	</head>
-	<body></body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Introduce</title>
+  </head>
+  <body>
+    <h1>김하늘의 자기소개입니다.</h1>
+    <p>안녕하세요! 저는 프론트엔드 개발자를 꿈꾸는 김하늘입니다. 만나서 반갑습니다.</p>
```
Comment (imparang) [inline index.html:10]: Could you use tags that emphasize the word 프론트엔드 개발자 within this statements?

**Review by imparang — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (lazy-sky) [reply index.html:11]: ok i'll fix it

File: index.html (hunk 2 above)
Comment (lazy-sky) [reply index.html:10]: this too.

**Review by imparang — APPROVED** [review]

**Pull request merged**